app/views/lessons/show.html.erb

```diff
@@ -26,14 +26,26 @@
 <div class="lesson-functions">
   <div class="container">
     <div class="card-main">
-      <h1 class="lesson-functions__title"><%= @lesson.course.title %></h1>
-      <h3 class="lesson-functions__sub-title accent bold"><%= @lesson.title %></h3>
+      <h1 class="lesson-functions__title">
+        <%= @lesson.course.title %>
+      </h1>
+
+      <h3 class="lesson-functions__sub-title accent bold">
+        <%= @lesson.title %>
+      </h3>
+
       <div class="lesson-button-group">
         <%= render 'lesson_buttons', lesson: @lesson, course: @lesson.course, user: @user %>
       </div>
 
       <% if @projects && ProjectSubmissionFeature.enabled? %>
-        <%= render 'submissions' %>
+        <div id="submissions">
```

Comment:
Do we need this div with the submissions id?  it doesn't seem to serve any purpose other than being used a selector for the js. If a selector is all thats needed here can we use the `submissions-container` instead please?